docs/cli.md

```diff
@@ -8,25 +8,29 @@ last_update:
 license: 'This software is licensed under the Apache License version 2.'
 ---
 
-This document provides guidance on managing metadata within Apache Gravitino using the Command Line Interface (CLI). The CLI offers a terminal based alternative to using code or the REST interface for metadata management.
+This document provides a guidance on managing metadata within Apache Gravitino using
+the Command Line Interface (CLI). The CLI offers a terminal based alternative to
+using code or the REST API for metadata management.
```

Comment:
Please don't break up sentences, this stops some IDE tools like grammar checkers from working and makes it easier for errors to slip into the text.

Reply:
Em... Let me try articulate the reasons behind this kind of changes.

- We'd better treat docs as code. That means we write them and then track whatever
  changes in the future.
- The GIT tool compare changes on a line by line basis. When we have a long line of
  text, GIT can only tell you that this "line" has changed. We as human reviewers have
  to identify the actual change(s) manually. That means a painful experience like finding
  a needle fallen on the ground.
- The GitHub web UI works the same way. I have myself been tortured by this for many years.
- When we break long lines manually, appropriately, this line breaks are not impacting
  the rendered output (mainly HTML here).
- With manual line breaks, we ensure that every developer sees the same thing on his
  or her environment. Think about the tab size problem here.
- I am not quite sure how an IDE helps improve the documentation. Even if there is an
  IDE (say IDEA) can do a great job for something, we are not writing code/docs for
  that IDE. Human is not supposed to be hijacked by tools, right?
- I'm open to learn the draw backs of manually wrapped long lines. In some other
  communities, manually wrapping long lines is strongly encouraged.
- This does NOT mean we want to encourage people to break the long lines at a fix boundary.
  The recommended way is to break them WHEN APPROPRIATE. For example, we encourage
  breaking long lines when a statement ends, when there is a punctuation character,
  so on and so forth. We discourage breaking links, however, because it doesn't make
  a lot of senses.

Reply:
The GitHub web UI doesn't work the same way it highlights word changes within a line.

We are writing docs in an IDE, and IDE plugins such as grammar checkers help do that. They don't always work if you break sentences over lines.

In this project, I noticed that in the documentation that breaks up sentences,  errors are easily introduced in changes (as they are here).

Reply:
![image](https://github.com/user-attachments/assets/08fefb2b-6568-4e7e-a283-08ee5569ac3b)

Let's use the change I made as an example. GitHub failed to identify that this change is only about inserting several line breaks.

Reply:
That shows no changes other than the newlines or lines moved. This is a better example:
<img width="671" alt="image" src="https://github.com/user-attachments/assets/71f4ff63-674d-4c04-82b2-49ad5308cb19" />

Reply:
Here you can see two words have been changed